modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/SpringCodegen.java

```diff
@@ -35,6 +37,7 @@
     protected String responseWrapper = "";
     protected boolean useTags = false;
 	protected boolean useBeanValidation = true;
+	protected boolean useOptional = true;
```

Comment:
@huningd I'll change this to `false` as part of the formatting change.

cc @cbornet